std/utf.d

```diff
@@ -4325,10 +4325,22 @@ if (isSomeChar!C)
                     this.buff = buff;
                 }
 
+                static if (isBidirectionalRange!R)
+                {
+                    this(return R r, uint frontBuff, uint backBuff)
+                    {
+                        this.r = r;
+                        this.buff = frontBuff;
+                        this.backBuff = backBuff;
```

Comment:
Is this constructor needed? I don't see any calls to it. Also, it seems that it is not covered by tests.